pages/products.html

```diff
@@ -0,0 +1,383 @@
+
+                </div>
+            </div>
+            <!-- card without badge -->
+            <div class="card-vertical pos-rel card-container flex-items display-flex-column card-vertical-shadow">
+                <div class="image-container badge-container pos-rel ">
+                    <img src="./../assets/images/products/pow3.png" alt="product-image" class="img-responsive card-img">
+                </div>
+                <div class="card-text-container pos-rel flex-grow display-flex-column">
+                    <div class="text-container-title">
+                        <h6 class="product-title pos-rel">Blush Makeup Brush
+                            <button class="like-btn link-no-style">
+                                <span><i class="fas fa-heart wish-icon"></i></span>
+                            </button>
+                        </h6>
+
+                    </div>
+                    <div class="text-container-desc  flex-grow display-flex-column">
+                        <span class=" text-price-lighter"> Rs.1200
+                            <small class="discount-txt">(17% off)</small>
+                        </span>
+                        <pre class="rating-text">5<i class="fa fa-solid fa-star rating"></i> |15   
+                        </pre>
+                    </div>
+                    <div class="CTA-container">
+                        <button class="btn-outline btn-on-hover">
+                            <a href="/pages/cart.html" class="link-no-style">Add to Cart</a>
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+    </div>
+
+    <!-- footer -->
+    <div class="footer">
+        <div class="container">
+            <div class="row-container">
+                <div class="footer-col-1">
+                    <h3>Follow me</h3>
+                    <ul class="quick-links">
+                        <li><a href="" class="link">Twitter</a></li>
+                        <li><a href="" class="link">Github</a></li>
+                        <li><a href="" class="link">LinkedIn</a></li>
+                        <li><a href="" class="link">Instagram</a></li>
+                    </ul>
+                </div>
+                <div class="footer-col-2">
+                    <h3>Quick links</h3>
+                    <ul class="quick-links">
+                        <li><a href="./index.html" class="link">Home</a></li>
+                        <li><a href="./pages/products.html" class="link">Products</a></li>
+                        <li><a href="./pages/cart.html" class="link">My Cart</a></li>
+                        <li><a href="./pages/wishlist.html" class="link">My Wishlist</a></li>
+                    </ul>
+                </div>
+            </div>
+            <hr>
+            <p class="madeby">Made with
+                <span class="white-color">&lt;/&gt;</span> by Manjushree</p>
+        </div>
+    </div>
```

Comment:
Just one suggestion, check the functionality of the footer on this page. It overlaps the sidebar.